plugins/scio/src/main/scala/org/broadinstitute/monster/sbt/MonsterScioPipelinePlugin.scala

```diff
@@ -8,7 +8,7 @@ import sbt.Keys._
 object MonsterScioPipelinePlugin extends AutoPlugin {
   override def requires: Plugins = MonsterDockerPlugin
 
-  val ScioUtilsVersion = "1.2.1"
+  val ScioUtilsVersion = "1.3.0"
```

Comment:
Talked on Slack, but recording here too for other reviewers: We missed bumping the Apache Beam dependency in scio-utils. Once we make that bump, we'll need to publish a `1.3.1` and use that here.